support-frontend/assets/pages/weekly-subscription-landing/components/content/GiftOrPersonal.jsx

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import React from 'react';
+import { css } from '@emotion/core';
+import { space } from '@guardian/src-foundations';
+import { body, headline } from '@guardian/src-foundations/typography';
+import { LinkButton } from '@guardian/src-button';
+import { SvgArrowRightStraight } from '@guardian/src-icons';
+
+import { routes } from 'helpers/routes';
+
+type PropTypes = {|
+  orderIsAGift: boolean,
+|};
+
+const giftOrPersonal = css`
+  padding: ${space[3]}px ${space[3]}px ${space[12]}px;
+`;
+
+const giftOrPersonalCopy = css`
+  ${body.medium()};
+  margin-bottom: ${space[9]}px;
+`;
+
+const giftOrPersonalHeading = css`
+  ${headline.medium({ fontWeight: 'bold' })};
+`;
+
+function GiftOrPersonal({ orderIsAGift }: PropTypes) {
```

Comment:
Since this component is now used in digital subs and weekly, it might be worth abstracting it further and putting it in shared components.

Reply:
I think I'll leave this for now, since we haven't launched DS gifts yet, but we should definitely do that in the near future.